components/notability/commons/notability_checker.lua

```diff
@@ -102,7 +102,8 @@ end
 
 function NotabilityChecker._calculatePlayerNotability(player)
 	player = mw.ext.TeamLiquidIntegration.resolve_redirect(player)
-	local conditions = '[[players_p' .. tostring(1) .. '::' .. player .. ']]'
+	local conditions = '[[players_p' .. tostring(1) .. '::' .. player .. ']]' ..
```

Comment:
some wikis have different prefixes, maybe make this changeable

mazbe something like
```suggestion
	local conditions = '[[players_' .. config.playerPrefix .. tostring(1) .. '::' .. player .. ']]' ..
```

Reply:
Simply put, this is the prefix that should be used. This module is incentive to standardize.

Reply:
would be okay for me (sc2 atm doesn't use that lpdb field at all atm, so we will (when we switch it) use the default one)

the question is if we want to add legacy support for wikis that have to switch the prefix

Reply:
Not a question, we don't :)